src/Logging/Logging.Console/src/ConsoleLoggerOptions.cs

```diff
@@ -9,12 +9,17 @@ public class ConsoleLoggerOptions
         public bool DisableColors { get; set; }
 
         /// <summary>
-        /// Gets or sets value indicating the minimum level of messaged that would get written to  <c>Console.Error</c>.
+        /// Gets or sets log message format. Defaults to <c>ConsoleLoggerFormat.Default</c>.
```

Comment:
`<see>` should do the needful here with hyperlinks and all.

```suggestion
        /// Gets or sets log message format. Defaults to <see cref="ConsoleLoggerFormat.Default" />.
```